pkg/cli/root.go

```diff
@@ -408,13 +408,13 @@ func LoadConfiguration(config *config.Config, configPath string) error {
 	return nil
 }
 
-func isDefaultPolicyConfig(cfg *config.Config) bool {
+func isAnonymousPolicyConfig(cfg *config.Config) bool {
 	adminPolicy := cfg.AccessControl.AdminPolicy
 
-	log.Info().Msg("checking if default authorization is possible")
+	log.Info().Msg("checking if anonymous authorization is possible")
```

Comment:
`log.Info().Msg("checking if anonymous authorization is the only type of authorization policy configured")`